modules/mobile/spec/workers/pre_cache_appointments_job_spec.rb

```diff
@@ -78,96 +78,107 @@
           end
         end
       end
+    end
```

Comment:
I re-structured this test file slightly because I noticed that the 'with errors' context was embedded within the 'without errors' context. This is the problem with embedding too much. It becomes visually difficult to parse and leads to this type of error. So I decided to flatten the file into more top-level contexts.